docs/en/observability/logs-troubleshooting.asciidoc

```diff
@@ -13,9 +13,9 @@ If you don't have the required privileges to create an API key, you'll see the f
 ----
 User does not have permissions to create API key.
 
-Required cluster privileges are [`monitor`, `manage_own_api_key`] and 
-required index privileges are [`auto_configure`, `create_doc`] for 
-indices [`logs-*-*`, `metrics-*-*`], please add all required privileges 
+Required cluster privileges are [`monitor`, `manage_own_api_key`] and
+required index privileges are [`auto_configure`, `create_doc`] for
+indices [`logs-*-*`, `metrics-*-*`], please add all required privileges
```

Comment:
I didn't comment on this during my review since the change is whitespace only, but this is a comma splice and should probably be fixed at some point.

Reply:
This is the actual error message, so I'll reach out to a dev to update there before fixing.